pkg/agent/secondarynetwork/podwatch/sriov.go

```diff
@@ -193,3 +193,19 @@ func (pc *PodController) deleteSriovSecondaryInterface(interfaceConfig *interfac
 	pc.releaseSriovVFDeviceID(interfaceConfig.PodName, interfaceConfig.PodNamespace, interfaceConfig.IFDev)
 	return nil
 }
+
+func (pc *PodController) GetAssignedVFDeviceNum(podName, podNamespace string) int {
+	podKey := podKeyGet(podName, podNamespace)
+	obj, cacheFound := pc.vfDeviceIDUsageMap.Load(podKey)
```

Comment:
What's the behavior after an agent restart? Can `vfDeviceIDUsageMap` be empty even if the a VF is assigned to the Pod? Could that lead to an incorrect clean up if the Pod is deleted (CNI DEL)?

Reply:
Hi @antoninbas I have submitted a PR to handle the secondary interface store recovery after agent restarts, please take a look. https://github.com/antrea-io/antrea/pull/7240. Thanks.